draft-ietf-ppm-dap.md

```diff
@@ -1714,6 +1710,29 @@ both of which can violate privacy ({{replay-protection}}). Before committing to
 an output share, the Aggregators check whether its report ID has already been
 aggregated and whether the batch bucket being updated has been collected.
 
+### Eager Aggregation {#eager-aggregation}
+
+In general, aggregation cannot begin until the Collector specifies a query and
+an aggregation parameter. However, depending on the VDAF and batch mode in use,
+it is often possible to begin aggregation as soon as reports arrive.
+
+For example, Prio3 has just one valid aggregation parameter (the empty string),
+and so allows for eager aggregation. Both the time-interval and leader-selected
+batch modes defined in this document ({{batch-modes}}) allow for eager
+aggregation, but future batch modes might preclude it.
+
+If the VDAF in use does have a non-trivial aggregation parameter, there might be
+some way the Aggregators can anticipate the parameter the Collector will choose
+and begin aggregation. For example, when using Poplar1 ({{Section 8 of !VDAF}}),
+the Collector and Aggregators might agree ahead of time on the set of candidate
+prefixes to use.
```

Comment:
```suggestion
prefixes to use. However this depends on the application.
```

Reply:
Sure, I added the word "application" earlier in the paragraph.